packages/build/src/plugins/mutations.js

```diff
@@ -62,7 +62,6 @@ const MUTABLE_PROPS = {
   'build.publish': { lastEvent: 'onPostBuild' },
   'build.services': { lastEvent: 'onPostBuild' },
   'build.services.*': { lastEvent: 'onPostBuild' },
-  edge_handlers: { lastEvent: 'onPostBuild' },
```

Comment:
This removes support for mutating `edge_handlers` in plugins for the moment.

The reason is because we have two properties:
  - `build.edge_handlers`: location of the edge handlers directory
  - `edge_handlers`: edge handlers configuration

(Side note: I am wondering if this might be confusing to some users, who might wonder which is which.)

The problem is that `context.*` properties target properties either at the top level or in `build.*`. So with `edge_handlers`, it is ambiguous which is which. We could distinguish by checking the type of the value, but we might want to do this in a follow-up PR, so I'm removing support for the moment.